spot-client/src/spot-tv/ui/views/home.js

```diff
@@ -189,7 +189,7 @@ export class Home extends React.Component {
      */
     _onOpenRemote() {
         const baseUrl = windowHandler.getBaseUrl();
-        const url = `${baseUrl}#/?code=${this.props.joinCode.toUpperCase()}`;
+        const url = `${baseUrl}/${this.props.joinCode.toUpperCase()}`;
```

Comment:
We should hide this one day in some kind of routing helper. Did you happen to find a decent place to shove it?

Reply:
No, but routing helper sounds good to me. Do you want me to create one now ?

Reply:
Nah, not for this PR.